File: backend/ws_wa_store_ccadmin/src/main/java/com/ccadmin/app/sale/model/entity/SaleChannelEntity.java
package com.ccadmin.app.sale.model.entity;

import com.ccadmin.app.shared.model.entity.AuditTableEntity;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import java.io.Serializable;

@Entity
@Table(name = "sale_channel")
public class SaleChannelEntity extends AuditTableEntity implements Serializable {

    @Id
    public String SaleCod;
    public String ChannelCod;

    public SaleChannelEntity() {
    }
}

File: frontend/web_ccadmin_lt/src/app/enterprise/sale/model/entity/SaleChannelEntity.ts
import { AuditTableEntity } from 'src/app/enterprise/shared/model/entity/AuditTableEntity';
import { SaleConstants } from '../constants/SaleConstants';

export class SaleChannelEntity extends AuditTableEntity {
  public SaleCod: string = '';
  public ChannelCod: string = SaleConstants.COMMERCIAL_CHANNEL_IN_PERSON;

  constructor() {
    super();
  }
}
